doc: BT service library docs clean up (#10975)
nrfconnect/sdk-nrf: doc/nrf/libraries/bluetooth_services/services/fast_pair.rst

* :kconfig:option:`CONFIG_BT_CTLR_DATA_LENGTH_MAX`

.. tip::

>    In case of the :ref:`nRF53 Series <ug_nrf53>` devices, this part of the configuration cannot be automatically updated for the network core and you must manually align it.

Reviewer: ```suggestion
   When using :ref:`nRF53 Series <ug_nrf53>` devices, this part of the configuration cannot be automatically updated for the network core and you must manually align it.
```
